fe/fe-core/src/main/java/com/starrocks/metric/MetricRepo.java

```diff
@@ -490,6 +490,16 @@ public Long getValue() {
         };
         STARROCKS_METRIC_REGISTER.addMetric(totalTabletCount);
 
+        // connections
+        GaugeMetric<Integer> connections = new GaugeMetric<Integer>(
```

Comment:
To following Prometheus metrics format, combine the outputs of starrocks_fe_connection_total and starrocks_fe_connection_total{user="xxx"} together.
That is you should put the `connection_total` into the `collectUserConnMetrics` and generate result according to requestParams.isCollectUserConnMetrics().